### ISSUE TITLE
Virtual DJ 8 Crashes on Patched Sur

### ISSUE THREAD

**Issue by abinthomas37** [BenSova/Patched-Sur#178]
I use Virtual DJ 8 for gigs and music production. Every time I force quit or even just simply close the VDJ8 app, my entire computer crashes and restarts. I've tried downloading older builds but for some reason, it doesn't work. The typical "Your computer has restarted because of a problem" message appears. Does anyone have any idea what could be causing the issue?

**Comment by BenSova** [comment]
It could just be a problem with Big Sur and Virtual DJ 8. Did you try to look and see if others have the same problem?

**Comment by abinthomas37** [comment]
https://www.virtualdj.com/forums/238877/VirtualDJ_Technical_Support/Virtual_DJ_2021is_crashing_my_mac_Big_Sur_11_1.html

This is the closest I could find^ I tried downloading the older builds and it still didn't work :/ 

Link to download VDJ: https://www.virtualdj.com/download/build.html
I used this to download older versions of the software but it still causes my macbook pro 2012 to crash and restart completely. The software, both Big Sur and VDJ8, work fine by themselves. The issue only arises when I quit VDJ and the macbook just restarts. 

P.S. Thanks for Patched Sur!
